.github/workflows/job_python_unit_tests.yml

```diff
@@ -216,6 +216,15 @@ jobs:
           TEST_DEVICE: CPU
           TEST_PRECISION: FP32
 
+      - name: PyTorch torch.export Layer Tests
+        if: ${{ fromJSON(inputs.affected-components).PyTorch_FE.test && runner.os != 'macOS' }}
+        run: |
+          python3 -m pytest ${LAYER_TESTS_INSTALL_DIR}/pytorch_tests -m precommit_torch_export --junitxml=${INSTALL_TEST_DIR}/TEST-pytorch.xml
+        env:
+          TEST_DEVICE: CPU
+          TEST_PRECISION: FP32
+          PYTORCH_TRACING_MODE: EXPORT
```

Comment:
please, add to Windows as well

Reply:
https://github.com/openvinotoolkit/openvino/blob/1a55740532e6d6ac53c556f95f94434276063ff7/.github/workflows/windows.yml#L487
Since regular tests are disabled now I don't think I should add these tests if I can't verify if they work or not

Reply:
ok, I've left a comment to that ticket to add these new tests once PyTorch tests will be enabled back on Windows